scripts-dev/docker_update_debian_changelog.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
```

Comment:
I'd say putting `-e` in the shebang isn't great style. If your script relies on `-e` being set, why not do so with `set -e` - which means it doesn't matter if it gets run as `./docker_update_debian_changelog.sh` or `bash docker_update_debian_changelog.sh`.

Not a big deal, just seems an odd choice to me.